pkg/api/v1/types.go

```diff
@@ -1816,6 +1816,7 @@ type ServiceSpec struct {
 	// API for compatibility until at least 8/20/2016.  It will be removed from
 	// any new API revisions.  If both deprecatedPublicIPs *and* externalIPs are
 	// set, deprecatedPublicIPs is used.
+	// +genconversion=false
```

Comment:
Actually - it depends :)
Basically, for generating client we use "second closest comment line", but e.g. for protobuf generation we are using the main comment, e.g.:
https://github.com/kubernetes/kubernetes/blob/master/pkg/api/unversioned/time.go#L30

And for member fields we don't have SecondClosestComment lines yet.

I've added a TODO and will fix that in a separate PR (this is just a single place for now, so shouldn't cause big user problems :))